core/iwasm/compilation/aot_llvm.c

```diff
@@ -327,9 +339,32 @@ aot_add_precheck_function(AOTCompContext *comp_ctx, LLVMModuleRef module,
     /*
      * load the value for this wrapped function from the stack_sizes array
      */
+    LLVMValueRef stack_sizes;
+    if (comp_ctx->is_indirect_mode) {
+        uint32 offset_u32 = get_inst_extra_offset(comp_ctx);
+        offset_u32 += offsetof(AOTModuleInstanceExtra, stack_sizes);
+        LLVMValueRef offset = I32_CONST(offset_u32);
+        if (!offset) {
+            goto fail;
+        }
+        LLVMValueRef stack_sizes_p =
```

Comment:
Had better declare the variables at the beginning of code piece `{ ... }`. And had better add empty lines if needed. We usually keep that coding style to make the code easier to read.
For example, there is no empty line from L342 to L402, a little intensive:)

Reply:
i generally feel lesser lines are easier to read. but ok. i will follow your style.

Reply:
done